### DISCUSSION TITLE
I have no access to github education pack

### DISCUSSION THREAD

**Discussion by lzy37ld** [community/community#122883] [GitHub Education]
### Hi everyone,

My student application is approved.

![image](https://github.com/community/community/assets/128442117/7e334a1f-2ba9-4620-a699-8fc5564f970b)
But I still could not access the GitHub educational pack like copilot.
It says
`We will notify you via email when your Experiences and offers become available to you.`

**Comment by 22coffeeboi** [comment]
Congratulations on getting your student application approved! It's great to hear that you've been approved for the GitHub Student Developer Pack.

It's not uncommon for there to be a delay between approval and gaining access to the benefits. The message you received indicates that you'll be notified via email when your experiences and offers become available to you. This likely means that GitHub is still processing your approval and setting up your access to the educational pack.

Usually, you should receive an email notification once your access has been granted. However, if you haven't received any updates after a reasonable amount of time, you might want to reach out to GitHub support for assistance. You can inquire about the status of your access and any potential issues that might be delaying it.

In the meantime, continue focusing on your studies and coding projects. Hopefully, you'll gain access to the GitHub Student Developer Pack and its benefits soon

**Comment by davevad93** [comment]
Hi @lzy37ld ,

Your student application has been approved, but you still have to wait 2-3 days for your benefits to become available. When the student benefits are finally available you will see the following message: 

`Congrats! Your request for @lzy37ld was approved and your benefits are now available.`

**Comment by rohit00900** [comment]
Hi everyone 
My GitHub education is approved too.
But I cannot access to my benefits
![Screenshot_2024-05-08-18-42-48-24](https://github.com/community/community/assets/78745210/cb0d21f9-bc24-452a-848c-e4bdd43da3dc)

![Screenshot_2024-05-08-18-42-48-24](https://github.com/community/community/assets/78745210/af55b2e3-46da-4476-a36b-25bbc4352e93)
 I don't know why this is happening 
Please tell me and give me my access

**Comment by rohit00900** [comment]
![Uploading Screenshot_2024-05-08-18-42-01-83.png…]()
This also

**Comment by rohit00900** [comment]
Hello sir please check my problem as soon as possible 

![Screenshot_2024-05-08-18-42-01-83](https://github.com/community/community/assets/78745210/251d774d-4a82-478b-b645-6cfb06442520)


See this also

**Reply by davevad93** [reply]
Hi @rohit00900 ,

I'm sorry, I'm just a user of GitHub as you are, so I don't know how to solve your issue. I advise you to contact [GitHub Education Support](https://support.github.com/request/education) in order to solve it.

**Comment by rohit00900** [comment]
Sony understand what's happening

**Comment by iterhim** [comment]
Hello everybody, I've got answer for our question on [this article](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
